code/object/waypoint.cpp

```diff
@@ -497,7 +497,6 @@ void waypoint_remove(waypoint *wpt)
 					break;
 				}
 			}
-			Assert(ii != Waypoint_lists.end());
```

Comment:
It appears that this (and the fredview changes) have slipped in from #2324.

Reply:
I saw that.  I rebased before I committed and I thought that fixed it, but I guess not.  I'll rebase to master once 2324 is merged.